### PR TITLE
Feature/folder name

### PR DESCRIPTION
**Pull request by redthor**
Hi there,

This PR is just to show you what I did. I don't expect it to be merged. I can submit another if you think some of it is worthwhile.

Basically I wanted to optionally pass a folder name with some depth, e.g.

echo '/home/user/repo.git library-package lib/packages/library-package' | tomono.sh

So in the monorepo you'll find ./lib/packages/library-package/*

I think I received an `ls` error at some point but otherwise it seems to work.
Thanks for the script.

### CODE REVIEW

File: tomono.sh
Comment (hraban) [inline tomono.sh]: to be safe, I'd prefer to always quote any variable in bash. `folder="$name"`

**Review by hraban — CHANGES_REQUESTED** [review]
Hi,

Thanks for your contribution. I think it could be worth merging.

I like the last commit, but the first two introduce a lot of unnecessary noise. Could you remove those and rebase?

And if you could, please add a little comment to the README explaining the new syntax?

Thanks again.

**Comment by redthor** [comment]
Hi there,
I've made the changes. I wasn't sure about the instruction for rebasing? I think the branch was already up to date with master if that's what you meant?

I've reverted the whitespace commits. I imagine you can squash the PR to remove all the extra commits...

I've made the README change and `"$name"` change.

Thanks.

**Comment by hraban** [comment]
Thanks. I've rebased your patches to avoid thrashing the history and paralysing git blame, but otherwise no changes.

To maintainers:

# Please don't merge this PR!

Merge #25 instead.

**Comment by redthor** [comment]
@hraban - quick git question: would squashing the merge commit avoid thrashing the history and paralysing git blame?

**Comment by hraban** [comment]
Yes, that's what I did in #25. Squash all commits except the readme, and move the last commit above the readme to squash it in, as well.

Assuming you're on branch feature/folder-name and your master is in line with ours; use `git rebase -i master` for that and move and mark lines appropriately. You'll want the `VISUAL` envvar set to your editor, e.g. 'vim' or 'emacs'.

**Comment by redthor** [comment]
Oh - I mean just use github to do it for you? e.g.

see this image

**Comment by hraban** [comment]
Ahh, gotcha. Yes, but I don’t have any write permissions on this repo :) I can only create prs, myself (left the company a few years back). The actual maintainers, God bless, come in once every few months and seem to merge everything without perhaps paying too much attention, so that could go wrong. :)

Gotta play the hand you’re dealt. In the end, the result is the same, though, no?

**Comment by redthor** [comment]
lol, ok @hraban :)

**Pull request closed without merging**
